src/js/options.ts

```diff
@@ -204,15 +204,52 @@ export interface BaseReactNativeOptions {
    */
   beforeScreenshot?: (event: Event, hint: EventHint) => boolean;
 
+  /**
+   * The sample rate for profiling
+   * 1.0 will profile all transactions and 0 will profile none.
+   */
+  profilesSampleRate?: number;
+
+  /**
+   * Track the app start time by adding measurements to the first route transaction. If there is no routing instrumentation
+   * an app start transaction will be started.
+   *
+   * Requires performance monitoring to be enabled.
+   *
+   * @default true
+   */
+  enableAppStartTracking?: boolean;
```

Comment:
Q: Why nullable if we already set a default value as true, will null behave the same as setting it to false?

Same question applies to the below nullable boolean with null support.

Reply:
To allow users to not specify the option in the init.